WordPress/Classes/ViewRelated/Blog/Blog Dashboard/BlogDashboardViewController.swift

```diff
@@ -79,6 +81,9 @@ final class BlogDashboardViewController: UIViewController {
         startAlertTimer()
 
         WPAnalytics.track(.mySiteDashboardShown)
+
+        complianceCoordinator = CompliancePopoverCoordinator(viewController: self)
+        complianceCoordinator?.presentIfNeeded()
```

Comment:
The `complianceCoordinator` property will remain retained even after the popover is dismissed. Not sure if something can be done to deallocate this property when the popover is dismissed.

Reply:
Yeah it is due to being the first coordinator and not down in a chain. It would get deallocated when the parent VC is gone but obviously in this case it won't be since it is one of the tabs. I'll try to find a not so complicated way to deallocate it.